Collaboration-tools/README.md

```diff
@@ -0,0 +1,71 @@
+# Team-Rio
+Repository For Team Rio
+
+> # VERSION CONTROL 
+
+# VERSION CONTROL WITH GIT AND GITHUB :gear:
+
+
+<p> Before we start, we recommend to use  vscode and the linux OS i.e any distribution for best developer experience. this doesnt mean windows or macOs won't work all the commands are universally compatible. 
+</p> 
+
+> ## This illustration is done on the ubuntu terminal. 
+
+> When creating new project on the local machine.
+*  launch terminal ``` ctrl + Alt + T ```
+*  move to the project directory ie. Desktop/Project  ```cd Desktop/Project ```
+*  or create a new project directory ```mkdir Project```
+*  then change into the project directory ```cd Project```
+
+*  initialise our folder to a git repository ```git init```
+*  create a file using vscode ```<new>``` 
+*  for those who wouldn't wish to leave terminal yet you could use "pico" ```pico <new>```
+*  add some content into the file for this case we would be using ```Hello Team Rio```
+*  we could at this point want to check the working tree ```git status```
+*  add a file to the working tree. ```git add -A```
+*  committ the change to our repository. ```git commit -m <new>```
+*  check the status. ```git status```
+*  create branches. ```git checkout -b <new branch name>```
```

Comment:
git checkout -b <branch> ===> creates and switches to the branch
git branch <branch> ===> creates the branch, does not switch to it

Reply:
true... noted